internal/controller/postgrescluster/pgbouncer.go

```diff
@@ -103,6 +103,13 @@ func (r *Reconciler) reconcilePGBouncerConfigMap(
 		(feature.Enabled(ctx, feature.OpenTelemetryLogs) || feature.Enabled(ctx, feature.OpenTelemetryMetrics)) {
 		err = collector.AddToConfigMap(ctx, otelConfig, configmap)
 	}
+	// If OTel logging is enabled and retentionPeriod is set, add logrotate config
+	if err == nil && otelConfig != nil && feature.Enabled(ctx, feature.OpenTelemetryLogs) &&
+		cluster.Spec.Instrumentation != nil && cluster.Spec.Instrumentation.Logs != nil &&
+		cluster.Spec.Instrumentation.Logs.RetentionPeriod != nil {
```

Comment:
🔧 We should always rotate so that logs do not fill the disk. Without a user-specified retention, use some "small" amount that allows PgBouncer log collection to work. 🤔 "small" is less than a day? 🤷🏻

Reply:
Maybe just 1 day?